supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/PipelinesApiConfigurationAttributes.scala

```diff
@@ -154,11 +157,12 @@ object PipelinesApiConfigurationAttributes {
         .getOrElse(LocalizationConfiguration(DefaultLocalizationAttempts).validNel)
 
     val vpcNetworkLabel: ErrorOr[Option[String]] = validate { backendConfig.getAs[String]("virtual-private-cloud.network-label-key") }
+    val vpcSubnetworkLabel: ErrorOr[Option[String]] = validate { backendConfig.getAs[String]("virtual-private-cloud.subnetwork-label-key") }
     val vpcAuth: ErrorOr[Option[String]] = validate { backendConfig.getAs[String]("virtual-private-cloud.auth")}
 
     val virtualPrivateCloudConfiguration: ErrorOr[Option[VirtualPrivateCloudConfiguration]] = {
-      (vpcNetworkLabel, vpcAuth) flatMapN  {
-        case (network, auth) => validateVPCConfig(network, auth)
+      (vpcNetworkLabel, vpcSubnetworkLabel, vpcAuth) flatMapN  {
+        case (network, subnetwork, auth) => validateVPCConfig(network, subnetwork, auth)
```

Comment:
you _might_ be able to say `... flatMapN validateVPCConfig` here

Reply:
TIL. Thank you!